app/dashboards/snap_application_dashboard.rb

```diff
@@ -3,25 +3,29 @@
 class SnapApplicationDashboard < Administrate::BaseDashboard
   ATTRIBUTE_TYPES = {
     id: Field::Number,
-    created_at: Field::DateTime,
-    updated_at: Field::DateTime,
-    signature: Field::String,
-    signed_at: Field::DateTime,
-    email: Field::String,
-    documents: Field::String,
-    phone_number: Field::String,
-    sms_subscribed: Field::Boolean,
-    consent_to_terms: Field::Boolean,
-    mailing_address_same_as_residential_address: Field::Boolean,
-    unstable_housing: Field::Boolean,
-    everyone_a_citizen: Field::Boolean,
+    additional_income: Field::Text,
```

Comment:
are we showing every field right now? If not, I'd probably opt to not include these array fields, since we mostly use them to determine which fields we need values for.Eg: if there is an additional income from pension, we get a value for `additional_income_pension` -- so showing both is kind of duplicative.

understand if we want to start with all fields are pare down later, too

Reply:
@jessieay - I *think* this was auto-generated, we just re-sorted them alphabetically. I'm OK to pare it down tho!

Reply:
I've pruned this down to only the fields we expose in the index, show and edit pages,